go.mod

```diff
@@ -12,18 +12,22 @@ require (
 	github.com/btcsuite/btcd v0.23.2
 	github.com/btcsuite/btcd/btcec/v2 v2.2.0
 	github.com/btcsuite/btcd/btcutil v1.1.0
+	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
```

Comment:
we must prevent this, ie. having 2 different versions of btcutil pkg in the main go.mod.

To avoid this, we must isolate the code requiring it in a dedicated submodule.

Reply:
As discussed to be considered in another PR